spec/factories/group.rb

```diff
@@ -5,8 +5,8 @@
     is_public false
 
     transient do
-      members_count 1
-      owners_count 1
+      members_count 0
+      owners_count 0
```

Comment:
Because otherwise it has unintended consequences for the layman: it creates new users automatically by default so if the `User.count` matters to your spec, it will fail.

Eg.: In my spec, I'd have to do
```
let!(:application) { FactoryGirl.create :doorkeeper_application, owner: user_1 }
FactoryGirl.create :application_user, user: user_1, application: application
FactoryGirl.create :application_user, user: user_1, application: application
```
to get the `User.count` I expect, just because some default in the `group` factory—which `doorkeeper_application` needs—which `application_user` needs.